src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+  cursor: pointer;
+
+  &__star {
+    background: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  & > &__star:nth-child(-n + 4) {
```

Comment:
Consider using css from the previous task to keep modifiers working

Reply:
done)